src/main/java/net/minecraftforge/client/model/generators/BlockModelBuilder.java

```diff
@@ -20,6 +20,7 @@
 package net.minecraftforge.client.model.generators;
 
 import net.minecraft.util.ResourceLocation;
+import net.minecraftforge.common.data.ExistingFileHelper;
```

Comment:
Unnecessary import here and a few times below

Reply:
It's not unnecessary...the class moved.